.github/workflows/deploy.yaml

```diff
@@ -4,24 +4,32 @@ on:
   push:
     branches:
     - main
+  pull_request:
+    branches: [ main ]
 
 jobs: 
   main:
     runs-on: ubuntu-latest
+    env:
+      IMAGE: quay.io/operator-framework/plain-provisioner:main
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '~1.17'
+
+      - name: Build container
+        run: make build-container
```

Comment:
Do we need to cache the build cache throughout this job?

Reply:
+1

Reply:
Just create a quick issue so I don't forgot about this again.